CONTRIBUTING.md

```diff
@@ -22,10 +22,11 @@ From a separate terminal:
 - `flutter run`
 
 From the packages/devtools directory:
-- `pub run webdev serve web`
+- `pub global activate webdev`
```

Comment:
can you add the step needed to make globally activated packages run as standalone executables or update the instructions to still include pub in the command?

Reply:
My understanding is that globally activated packages are available as standalone executables by default (assuming Dart is on the path). Is this not the case?

Reply:
It doesn't work for me.